flexget/_version.py

```diff
@@ -7,4 +7,4 @@
 The github actions release job will automatically strip the .dev for release,
 and update the version again for continued development.
 """
-__version__ = '3.8.8.dev'
+__version__ = '3.9.dev'
```

Comment:
3.9 is invalid on semver that broke the web interface.

This is from the browser console:

```
TypeError: Invalid Version: 3.9
    l semver.js:41
    exports compare.js:3
    exports gt.js:2
    Ot Version.tsx:25
    React 7
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
    e hooks.tsx:10
    u runtime.js:45
    _invoke runtime.js:271
    t runtime.js:97
    bt main.cce4d52f6988bfadab4f.js:1
    i main.cce4d52f6988bfadab4f.js:1
    promise callback*bt main.cce4d52f6988bfadab4f.js:1
    i main.cce4d52f6988bfadab4f.js:1
    e main.cce4d52f6988bfadab4f.js:1
    e main.cce4d52f6988bfadab4f.js:1
    yt hooks.tsx:7
    yt hooks.tsx:13
    React 2
    unstable_runWithPriority scheduler.production.min.js:18
    React 4
    unstable_runWithPriority scheduler.production.min.js:18
    React 5
    e Login.tsx:17
    u runtime.js:45
    _invoke runtime.js:271
    t runtime.js:97
    $ 14.18cf3ab07df2cff9c980.js:1
    i 14.18cf3ab07df2cff9c980.js:1
    promise callback*$ 14.18cf3ab07df2cff9c980.js:1
    i 14.18cf3ab07df2cff9c980.js:1
    e 14.18cf3ab07df2cff9c980.js:1
    e 14.18cf3ab07df2cff9c980.js:1
    default Login.tsx:14
    default Login.tsx:20
    React 2
    unstable_runWithPriority scheduler.production.min.js:18
    React 4
    unstable_runWithPriority scheduler.production.min.js:18
    React 2
    V scheduler.production.min.js:17
    onmessage scheduler.production.min.js:14
    EventHandlerNonNull* scheduler.production.min.js:13
    Webpack 14
```

Reply:
I just added a note in our version file so that we hopefully remember next time. A new release will come out tomorrow and fix the issue.